Integrations/integration-Office_EWS.yml

```diff
@@ -904,7 +912,7 @@ script:
         }
         fetchArgs.last_time = new Date(lastRun.lastTime).toISOString();
         fetchArgs['target-mailbox'] = getDefaultTargetMailbox();
-        if (params.folder) {
+        if (lastRun.folderId) {
```

Comment:
It means you set `folder` just after `folderId` was changed.

Reply:
No, last run is loaded from the server each time.
And stored at the flow.
So for the first time, i will load a lastRun that will not have a folderId and params will have it, then i will put it in, and will save.
The next time, they will have the same value, so lastRun will have a folderId and we are good

What am i missing?

Reply:
What I am missing is why should it solve the issue. Also today if params had it we would get into this if. So how does getting it from the last run instead of the param make it any better?

Reply:
The problem is that we checked the param object
But took the data from the lastParam object
Now hey are in sync

Reply:
ok. told you that I missed something.